fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/RegistryAttribute.java

```diff
@@ -0,0 +1,14 @@
+package net.fabricmc.fabric.api.event.registry;
+
+public enum RegistryAttribute {
```

Comment:
Hmm, can we use like a class than an enum in case mods add custom attributes

Reply:
Humm, possibly. Not too sure what the use would be / if they would need to be more than just a yes/no?